src/pages/CommitDetailPage/BotErrorBanner/BotErrorBanner.jsx

```diff
@@ -22,57 +22,90 @@ function useProviderSetting() {
   })
 
   const provider = providerToName(paramProvider)
-  const hasGhApp = accountDetails?.integrationId
+  const ghWithApp =
+    accountDetails?.integrationId && provider === ProvidersEnum.Github
 
-  const ghWithNoApp = provider === ProvidersEnum.Github && !hasGhApp
-  const bbOrhasGhApp =
-    provider === ProvidersEnum.BitBucket ||
-    (hasGhApp && provider === ProvidersEnum.Github)
+  const ghWithNoApp =
+    !accountDetails?.integrationId && provider === ProvidersEnum.Github
+
+  const bbProvider = provider === ProvidersEnum.BitBucket
   const glProvider = provider === ProvidersEnum.Gitlab
 
-  return { ghWithNoApp, bbOrhasGhApp, glProvider }
+  return { ghWithNoApp, bbProvider, glProvider, ghWithApp }
 }
 
-function BotErrorBanner({ botErrorsCount }) {
-  const { ghWithNoApp, bbOrhasGhApp, glProvider } = useProviderSetting()
+const BotErrorContent = () => {
+  const { ghWithNoApp, bbProvider, glProvider, ghWithApp } =
+    useProviderSetting()
+
+  if (ghWithNoApp) {
+    return (
+      <div className="gap-1 lg:flex lg:w-max">
+        <span>
+          The bot posts the coverage report comment on pull requests. If
+          you&apos;re using GitHub, the best way to integrate with Codecov.io is
+          to Install
+        </span>
+        <A to={{ pageName: 'codecovGithubApp' }}> Codecov&apos;s GitHub App.</A>
+      </div>
```

Comment:
Wanna switch this to a `<p>` 👀

Reply:
Also just noticed the `lg:w-max`, that is not a valid tailwind class the `w-max` specifically. See [here](https://tailwindcss.com/docs/max-width#reading-width).

Reply:
nice catch, i think we can safely get rid of all classes in this div